Code/Editor/Plugins/EditorAssetImporter/ImporterRootDisplay.h

```diff
@@ -40,25 +42,57 @@ namespace AZ
 
         }
     }
-}
+} // namespace AZ
 
 namespace Ui
 {
     class ImporterRootDisplay;
-}
+} // namespace UI
 
-class ImporterRootDisplay 
+//! Python interface for the scene importer root display settings
+class SceneSettingsRootDisplayPythonRequests
```

Comment:
normally we do not specify a bus as a Python only bus, maybe a "Script" bus or just SceneSettingsRootDisplayRequests?

Note that C++ and Lua can also call this bus.

Reply:
Good point, I've renamed this